udata_front/theme/gouvfr/datagouv-components/src/components/InformationPanel/InformationPanel.vue

```diff
@@ -0,0 +1,196 @@
+                    </div>
+                </div>
+            </div>
+        </div>
+    </article>
+    <article v-if="props.dataset?.harvest">
+        <header
+            class="fr-grid-row fr-grid-row--middle fr-pb-3w fr-mb-3w border-bottom border-default-grey"
+            :class="{'border-bottom': !harvestExpanded}"
+        >
+            <div class="fr-col">
+                <h2 class="subtitle subtitle--uppercase fr-m-0">{{ $t('Harvest') }}</h2>
+            </div>
+            <div class="fr-col-auto">
+                <button
+                    @click="expand"
+                    role="button"
+                    :aria-expanded="harvestExpanded"
+                    class="fr-btn fr-btn--tertiary-no-outline fr-btn--icon-left fr-btn--secondary-grey-500"
+                    :class="{'fr-icon-arrow-up-s-line': harvestExpanded, 'fr-icon-arrow-down-s-line': !harvestExpanded}"
+                >
+                    <template v-if="harvestExpanded">
+                        {{ $t('Close details') }}
+                    </template>
+                    <template v-else>
+                        {{ $t('See harvest') }}
+                    </template>
+                </button>
+            </div>
+        </header>
+        <div
+            class="accordion-content"
+            ref="harvestRef"
+        >
+            <div class="fr-pb-3w fr-mb-3w border-bottom border-default-grey">
+                <div class="fr-grid-row fr-grid-row--gutters fr-text--sm fr-m-0">
+                    <div v-for="(value, key) in props.dataset?.harvest" :key="key" class="fr-col-12 fr-col-sm-6 fr-col-md-4">
+                        <h3 class="subtitle fr-mb-1v">{{ key }}</h3>
+                        <p class="fr-text--sm fr-m-0 text-mention-grey ">{{ value }}</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </article>
+</template>
+<script setup lang="ts">
+import { ref, onMounted } from 'vue';
+import type { Dataset, DatasetV2 } from "../../types/datasets";
+import CopyButton from "../CopyButton/CopyButton.vue";
+import { toggleAccordion } from "../../helpers/toggleAccordion";
+import { formatDate } from '../../helpers/index';
+import { getGranularity, fetchGranularities } from '../../api/granularity';
+import { getFrequencies, fetchFrequencies } from '../../api/frequency';
+import useOEmbed from '../../composables/useEmbed'
+import type { Frequencies } from '../../types/frequency';
+import type { Granularities } from '../../types/granularity';
+import type { License } from '../../types/licenses';
+const props = defineProps<{
+    dataset: DatasetV2 | Dataset,
+    license: License
+}>();
+const embedText = useOEmbed('dataset', props.dataset.id);
+const extrasExpanded = ref(false);
+const extrasRef = ref(null);
+const granularities = ref<Granularities>([]);
+const frequencies = ref<Frequencies>([]);
+const harvestExpanded = ref(false);
+const harvestRef = ref(null);
```

Comment:
You can type the ref with the Div Element | null. Like you do below for textAreaRef